app/javascript/packs/admin_skills/skills.jsx

```diff
@@ -17,11 +17,11 @@ class Skills extends React.Component {
   }
 
   render() {
-    const skills = Object.keys(this.props.skills).map(skill => (
+    const skills = this.props.skills.map(skill => (
       <Skill
-        id={this.props.skills[skill]}
-        key={this.props.skills[skill].toString()}
-        skill={skill}
+        id={skill.id}
+        key={skill.id}
+        skill={skill.name}
```

Comment:
it is a good practice to name keys as they are. If we are talking about skill name, it should be a skillName.